internal/cli/atlas/clusters/indexes/create.go

```diff
@@ -116,8 +155,20 @@ func CreateBuilder() *cobra.Command {
   
   # Create a compound index named property_room_bedrooms on the
   listings collection of the realestate database:
-  atlas clusters indexes create property_room_bedrooms --clusterName Cluster0 --collection listings --db realestate --key property_type:1 --key room_type:1 --key bedrooms:1`,
-		PreRunE: func(cmd *cobra.Command, _ []string) error {
+  atlas clusters indexes create property_room_bedrooms --clusterName Cluster0 --collection listings --db realestate --key property_type:1 --key room_type:1 --key bedrooms:1
+
+  # Create an index named my_index from a JSON configuration file named myfile.json:
+  atlas clusters indexes create my_index --clusterName Cluster0 --file file.json`,
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			if opts.filename == "" {
+				_ = cmd.MarkFlagRequired(flag.Database)
+				_ = cmd.MarkFlagRequired(flag.Collection)
+				_ = cmd.MarkFlagRequired(flag.Key)
+				if len(args) == 0 {
+					return errors.New("index name missing")
```

Comment:
name is optional and derived from the keys is not provided

Reply:
I missed this. Thanks! updating.....

Reply:
updated